src/controllers/getURL.js

```diff
@@ -0,0 +1,18 @@
+import Url from '../models/UrlShorten';
+
+const getUrlAndEditCount = async (req, res) => {
```

Comment:
Can we rename this function to `updateUrlClickCount`

Reply:
The thing is that two things are happening in the route. We are updating the count and then we are redirecting the trimmed url to the original url, that was the reason for the naming.

Reply:
Okay